Kconfig.zephyr

```diff
@@ -845,7 +845,7 @@ endmenu
 
 config LEGACY_INCLUDE_PATH
 	bool "Allow for the legacy include paths (without the zephyr/ prefix)"
-	default y
+	default n
```

Comment:
`default n` is redundant and should be removed.

Reply:
Right! Dropped.